sdk/maps/Azure.Maps.Rendering/src/Generated/RenderingModelFactory.cs

```diff
@@ -11,7 +11,7 @@
 namespace Azure.Maps.Rendering
 {
     /// <summary> Model factory for models. </summary>
-    public static partial class MapsRenderingModelFactory
+    public static partial class RenderingModelFactory
```

Comment:
changed this back despite this package is preview because this name reads quite generic.